deepchecks/nlp/utils/text_properties.py

```diff
@@ -472,6 +486,14 @@ class TextProperty(TypedDict):
 )
 
 
+ALL_PROPERTIES: Tuple[TextProperty, ...] = (
```

Comment:
I don't understand the logic here - why treat these 4 new properties differently than all the rest? Why keep a Defaults list and an "all" list?